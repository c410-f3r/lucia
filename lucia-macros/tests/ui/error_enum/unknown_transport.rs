#![allow(incomplete_features)]
#![feature(async_fn_in_trait, impl_trait_projections)]

#[lucia_macros::pkg(api(Foo), data_format(json), transport(dasdasd))]
mod pkg {
  #[pkg::req_data]
  struct Req;

  #[pkg::res_data]
  struct Res;
}

fn main() {
}
