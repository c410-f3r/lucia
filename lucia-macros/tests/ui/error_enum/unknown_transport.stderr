error: Unknown transport.
 --> tests/ui/error_enum/unknown_transport.rs:4:60
  |
4 | #[lucia_macros::pkg(api(Foo), data_format(json), transport(dasdasd))]
  |                                                            ^^^^^^^
